Library/Formula/natalie.rb

```diff
@@ -0,0 +1,22 @@
+class Natalie < Formula
+  homepage "https://github.com/krzyzanowskim/Natalie"
+  url "https://github.com/krzyzanowskim/Natalie/archive/0.1.tar.gz"
+  sha256 "943dab1b7ac16555ec3047983a3c397f7ebe7559d3c098c02a6ee6d8e966655e"
+  head "https://github.com/krzyzanowskim/Natalie.git"
+
+  depends_on :xcode => "6.0"
+
+  def install
+    bin.install "natalie.swift"
+    share.install Dir["NatalieExample"] unless File.exists? (share/"NatalieExample")
```

Comment:
You can remove the conditional; `share` is inside the private prefix, it is guaranteed that it will not exist when this code is executed.